### ISSUE TITLE
[Plant Growth Package] Rework critical and non-critical packages

### ISSUE THREAD

**Issue by HLWeil** [nfdi4plants/arc-validate-package-registry#65]
The checked terms in the [plant growth validation package](https://github.com/nfdi4plants/arc-validate-package-registry/blob/main/StagingArea/plant-growth/plant-growth%401.0.0.fsx) are the ones from the [plant growth condition template](https://github.com/nfdi4plants/Swate-templates/blob/main/templates/dataplant/Plant_growth_conditions.xlsx).

Now the question is which should be critical (required) and which should be non-critical? For the first version I based it on some ARCs but this is pretty arbitrary atm.

Any ideas or people to include in the discussion? @Brilator

**Comment by Brilator** [comment]
The [Plant_growth_conditions.xlsx](https://github.com/nfdi4plants/Swate-templates/blob/main/templates/dataplant/Plant_growth_conditions.xlsx) template is the oldest template – from a time when we still had a very different to null understanding and tooling how metadata comes together :D (ontologies, swate, public repositories, etc.). I would not generally recommend to use this as a validation show-case. 

> which should be critical (required) and which should be non-critical

I'd recommend that we as DataPLANT do not reinvent those checklists of what is required / optional, but rather build this based on existing templates from communities / repositories

Maybe, `Swate-templates/templates/dataplant/ArrayExpress_-_Plant_sample.xlsx` or `Swate-templates/templates/dataplant/ENA/ENA_-_Plant_sample_-_extension.xlsx` would be better examples and already have requirements. 

I'm sure @StellaEggels can add to this.

**Comment by StellaEggels** [comment]
We did update the plant growth conditions templates not too long ago, but I agree that for defining required/optional terms it would make sense to base it on the repositories' definitions. ENA- Plant sample contains the required terms, ENA-Plant sample- extension, the recommended ones.

**Comment by HLWeil** [comment]
Yeah it probably makes more sense to have hard contraints only for end-point specific validation packages.

But IMO we should we still keep generic validation packages with mostly recommended terms around to encourage users to share some terms they could otherwise have forgotten about.

**Comment by Brilator** [comment]
Add to yaml: First critical term is, whether a protocol type "plant growth protocol" exists 

```yaml
|> Seq.exists (fun token -> 
    token.Name = "ProtocolType" 
    && 
    //Param.getValueAsTerm token = (CvTerm.create("DPBO:1000164","plant growth protocol","DPBO"))
    (Param.getValueAsTerm token).Name = "plant growth protocol"
)
 ```
